src/test/resources/features/delete_employee.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Delete Employee
+  The microservice provides a service to delete an employee based on ID,
+  The request is based on employee ID
+  we delete the particular employee
+
+  Scenario:
+    Given there is a delete employee request based on ID
+    When there exists an employee on that ID
```

Comment:
A good scenario would be 

Given Employees exist in the database
When I send a delete employee request with the employee ID
Then the employee information should be deleted

Reply:
The idea of BDD is that the feature scenarios that you write should be understood by everyone in the organization be it the developers, QA, BA or even anyone outside the project

Reply:
acknowledged - 8aea984